### PR TITLE
Allow to add tags with Passive Rules

### PR DESCRIPTION
**Pull request by denniskniep**

### CODE REVIEW

File: src/org/zaproxy/zap/extension/pscan/scanner/ScriptsPassiveScanner.java [hunk 1]
```diff
@@ -131,7 +131,11 @@ public void raiseAlert(int risk, int confidence, String name, String description
 
 		this.parent.raiseAlert(currentHRefId, alert);
 	}
-
+
```
Comment (thc202) [inline src/org/zaproxy/zap/extension/pscan/scanner/ScriptsPassiveScanner.java]: This indentation change could be reverted.

**Comment by thc202** [comment]
Thanks!

**Comment by denniskniep** [comment]
No problem. But there is unfortunately another indentation change in the new file `src/scripts/templates/passive/Passive default template.js`
I will fix this quickly.

**Comment by kingthorin** [comment]
Thanks @denniskniep

**Pull request merged**